### PR TITLE
Cleaned Up the Wiser Repo

### PR DESCRIPTION
**Pull request by Safranchik**
Cleaned up the wiser repo and added necessary functions to train the generative models on wiser/gen

Created an eval module for the eval.py class on wiser/eval, which now contains the class with the evaluation functions for LFs

Created the necessary __init__.py files

Modified the setup.py file to register the viewer HTML and JS files

### CODE REVIEW

File: wiser/gen/util.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        else:
+            sort = sorted(votes.keys(), key=lambda x: votes[x], reverse=True)
+            first, second = sort[0:2]
+            if votes[first] == votes[second]:
+                maj_vote[i] = treat_tie_as
+            else:
+                maj_vote[i] = first
+
+
+def get_gen_model_inputs(instances, label_to_ix):
```
Comment (stephenbach) [inline wiser/gen/util.py]: And get_label_model_inputs

File: wiser/gen/util.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                best_acc_prior = acc_prior
+                best_balance_prior = balance_prior
+
+    best_model = model_constructor(best_acc_prior, best_balance_prior)
+    best_model.load_state_dict(best_params)
+
+    return best_model, best_p, best_r, best_f1
+
+
+def train_generative_model(model, train_data, dev_data, epochs,
```
[the remark this answers is not in the document]
Comment (Safranchik) [reply wiser/gen/util.py]: Renaming the module "labelmodel" might get users confused with the  "labelmodels" package (https://github.com/BatsResearch/labelmodels). How about renaming "gen" to "generative" or "generative_models"? We could alternatively move this file to the labelmodels repo itself. I personally think the last option is the cleanest one.

**Comment by stephenbach** [comment]
Ok, let’s change the name to “generative.” And let’s make those two function names consistent, either “gen” or “generative.”

**Comment by Safranchik** [comment]
Done. Renamed the module to generative, and changed the "gen" to "generative" in the method name.

**Pull request merged**